src/content/utils.js

```diff
@@ -103,4 +103,25 @@ function algorithmToTag(item, showTagsInEnglish) {
             algorithm_id: item.algorithm_id
         }
     }
+}
+
+function getExpPointFromLevel(level){
+    const levelData = [320, 480, 912, 1642, 2791, 4465, 6698, 9913, 14472, 20840, 29593, 
+                       41431, 57588, 79472, 108877, 148072, 200638, 270862, 364309, 488174, 
+                       651712, 866777, 1148479, 1515993, 1993531, 2611525, 3408040, 4430452,
+                       5737436, 7401292, 9510661]
```

Comment:
Level experience points are subject to change, it should NOT be declared as constants. Please use https://api.solved.ac/exp_table.php

Reply:
If we had an api document(or just list), it would be great to develop some of functions.

Reply:
I will document APIs in the future, but it's a shame that I'm currently too busy to make and publish documented APIs.